manuscripts/method/06.methods.md

```diff
@@ -18,38 +18,38 @@ During model initialization, the pairwise distance between all peptides in the d
 
 $$w=\frac{1}{n} \left(P \cdot v \right)$$
 
-Where $P$ is the $n \times n$ distance matrix, $n$ is the number of peptides in the dataset, $v$ is the probability of each peptide being assigned to the cluster of interest, and $w$ is the log-probabilities of cluster assignment.
+Where $P$ is the $n \times n$ distance matrix, $n$ is the number of peptides in the dataset, $v$ is the probability of each peptide being assigned to the cluster of interest, and $w$ is the log-probabilities of cluster assignment. 
 
 #### Binomial enrichment
 
-We alternatively used a binomial enrichment model for the sequence representation of a cluster based on earlier work [@PMID:16273072]. Upon model initialization, a background matrix $G$ was created with a position-specific scoring matrix of all the sequences together. Next, an $i \times j \times k$ data tensor $T$ was created where $i$ is the number of peptides, $j$ is the number of amino acid possibilities, and $k$ is the position relative to the phosphorylation site. This tensor contained 1 where an amino acid was present for that position and peptide, and 0 elsewhere.
+We alternatively used a binomial enrichment model for the sequence representation of a cluster based on earlier work [@PMID:16273072]. Upon model initialization, a background matrix $i \times j \times k$ was created with a position-specific scoring matrix of all the sequences together. Next, a $T$ data tensor $i$ was created where $j$ is the number of peptides, $k$ is the number of amino acid possibilities, and $k$ is the position relative to the phosphorylation site. This tensor contained 1 where an amino acid was present for that position and peptide, and 0 elsewhere.
```

Comment:
This doesn't look right...

Reply:
Yes, meant to add a comment. This is how it looks like in the word document. Let's fix it there.

Reply:
I'm not comfortable merging changes we know are wrong, just because they match the Word document.